tests/helpers/test_storage.py

```diff
@@ -30,6 +31,14 @@ def store(hass):
     assert data == MOCK_DATA
 
 
+async def test_overwriting(hass, store):
```

Comment:
This actually doesn't do anything because all writes for the storage helper are mocked in tests

Reply:
Correct, we try to avoid I/O as much as possible in unit tests as it's slow.

We should still test this functionality but it should test `util.save_json` directly, as that one is not mocked out.